Microsoft.NET.Build.Containers/CreateNewImageToolTask.cs

```diff
@@ -10,95 +10,8 @@ namespace Microsoft.NET.Build.Containers.Tasks;
 /// <summary>
 /// This task will shell out to the net7.0-targeted application for VS scenarios.
 /// </summary>
-public class CreateNewImage : ToolTask
+public partial class CreateNewImage : ToolTask
 {
-    /// <summary>
-    /// The path to the folder containing `containerize.dll`.
-    /// </summary>
-    [Required]
-    public string ContainerizeDirectory { get; set; }
-
-    [Required]
-    public string BaseRegistry { get; set; }
-
-    /// <summary>
-    /// The base image to pull.
-    /// Ex: dotnet/runtime
-    /// </summary>
-    [Required]
-    public string BaseImageName { get; set; }
-
-    /// <summary>
-    /// The base image tag.
-    /// Ex: 6.0
-    /// </summary>
-    [Required]
-    public string BaseImageTag { get; set; }
-
-    /// <summary>
-    /// The registry to push to.
-    /// </summary>
-    [Required]
-    public string OutputRegistry { get; set; }
-
-    /// <summary>
-    /// The name of the output image that will be pushed to the registry.
-    /// </summary>
-    [Required]
-    public string ImageName { get; set; }
-
-    /// <summary>
-    /// The tag to associate with the new image.
-    /// </summary>
-    public ITaskItem[] ImageTags { get; set; }
```

Comment:
This changed: on the other side it was a `string[]` and I kept it that way in the common interface.